documentation/source/user_section/tutorials/lesion-analysis/lesion-segmentation.rst

```diff
@@ -0,0 +1,17 @@
+.. _lesion-segmentation:
+
+Lesion segmentation in spinal cord injury (SCI) or multiple sclerosis (MS)
+##########################################################################
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/lesion-analysis/sciseg.png
```

Comment:
Maybe change font to be coherent with the other tutorials

Reply:
Good point! I change fonts to Arial at https://github.com/spinalcordtoolbox/doc-figures/commit/ca2dc0a0634cc68871abf97be2e574b002645bf4. What do you think?